### PR TITLE
Update to Polymer 1.1

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
   "homepage": "https://github.com/PolymerElements/paper-toggle-button",
   "ignore": [],
   "dependencies": {
+    "polymer": "Polymer/polymer#^1.1.0",
```
Comment (notwaldorf) [inline bower.json:23]: This isn't needed (`#^1.0.0` should pull in `^1.1.0` as well)

File: bower.json [hunk 2]
```diff
@@ -20,17 +20,17 @@
...
     "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0"
+    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0"
   },
   "devDependencies": {
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.11",
```
Comment (notwaldorf) [inline bower.json]: This isn't needed. `^0.7.0` pulls in `0.7.11` as well

File: paper-toggle-button.html [hunk 3]
```diff
@@ -59,120 +162,118 @@
     </div>
 
   </template>
+</dom-module>
```
Comment (notwaldorf) [inline paper-toggle-button.html]: There isn't an advantage to pulling the script out of the `<dom-module>`, and it makes this patch much scarier than it is, so please don't do that :)

**Comment by notwaldorf** [comment]
@abdonrd I left a couple of comments. You'll also have to do a rebase, since I've merged something to master this morning. Finally, when you're done, would you mind squashing your commits? Thanks! :balloon:

**Comment by notwaldorf** [comment]
Thanks so much for the style cleanup, btw!

**Comment by abdonrd** [comment]
Updated!

Your welcome, @notwaldorf! :smile:

**Comment by abdonrd** [comment]
Updated with only one commit! :)

**Comment by notwaldorf** [comment]
I think the `webcomponents.js` version is still iffy in bower.json. Other than that, this looks good! :)

**Comment by abdonrd** [comment]
Sorry @notwaldorf, problem squashing the commits. :sweat_smile: Fixed! :smile:

**Comment by notwaldorf** [comment]
Thanks! Sorry for these being so tedious :(

**Pull request merged**